CRM/Report/Form/Contribute/Summary.php

```diff
@@ -514,6 +532,15 @@ public function from($entity = NULL) {
           ON ({$this->_aliases['civicrm_batch']}.entity_id = eft.financial_trxn_id
           AND {$this->_aliases['civicrm_batch']}.entity_table = 'civicrm_financial_trxn')";
     }
+
+    // for credit card type
+    if ($this->isTableSelected('civicrm_financial_trxn')) {
+      $this->_from .= "\n LEFT JOIN civicrm_entity_financial_trxn eftcc
```

Comment:
Does this have any effect on readability in the developer tab? If it doesn't then I'll get rid of it. There are many places in the current SQL where line breaks are present though.